services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategy.java

```diff
@@ -22,14 +22,24 @@
   private static final Logger logger = LoggerFactory.getLogger(ProdPpacIosRateLimitStrategy.class);
   static final int VALIDITY_IN_HOURS = 23;
 
+  private final Clock clock;
+
+  public ProdPpacIosRateLimitStrategy() {
+    this.clock = Clock.systemUTC();
+  }
+
+  public ProdPpacIosRateLimitStrategy(Clock clock) {
```

Comment:
Maybe we need a Configuration class that injects for prod (based on a profile or a name) the no-arg constructor and for tests it injects the clock constructor. What do you think?

Reply:
We can do that if you want – but I see no benefits over this approach but added complexity.

Reply:
Agree - based on the other comment